priv/clickhouse_repo/migrations/20221130162330_remove_campaign_id.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Plausible.ClickhouseRepo.Migrations.RemoveCampaignId do
+  use Ecto.Migration
+
+  def change do
+    alter table(:events) do
+      remove :campaign_id
```

Comment:
I suppose there is no operation to change field type?

Reply:
I was not sure how the `UInt64` -> `:string` conversion would behave (i.e. if `0` would be converted to an empty string), so I thought removing and recreating fields would be a safer option here.

Reply:
Sure, better safe than sorry. At this point of development it's probably better to do a full remove & add cycle